merlin/systems/triton/__init__.py

```diff
@@ -34,42 +34,103 @@
 )
 
 
-def convert_df_to_triton_input(column_names, batch, input_class=grpcclient.InferInput):
-    columns = [(col, batch[col]) for col in column_names]
-    inputs = []
-    for i, (name, col) in enumerate(columns):
-        if is_list_dtype(col):
+def convert_df_to_triton_input(schema, batch, input_class=grpcclient.InferInput, dtype="int32"):
```

Comment:
We need the schema here to get the column shapes, but it's handy in a few other ways too (like checking `is_ragged`.)